crates/cli/commands/src/init_state/without_evm.rs

```diff
@@ -33,7 +33,7 @@ pub fn setup_without_evm<Provider>(
 where
     Provider: StaticFileProviderFactory
         + StageCheckpointWriter
-        + BlockWriter<Body: reth_node_api::BlockBody>,
+        + BlockWriter<Block: reth_node_api::Block<Header = reth_primitives::Header>>,
```

Comment:
```suggestion
        + BlockWriter<BlockHeader = reth_primitives::Header>,
```

Reply:
`BlockWriter` doesn't have this AT

Reply:
well seems like it should then. anywhere in there impl that a transaction is also used? we don't want to be using fully qualified syntax or yet another adapter there for no apparent reason. just use the AT `Primitives: NodePrimitives` for now. `BlockPrimitives` would make most sense.